src/action.ts

```diff
@@ -38,27 +39,49 @@ async function downloadUpx(): Promise<string> {
   throw 'unsupported OS'
 }
 
+function resolve(input: string): string[] {
+  return input
+    .split(/\r?\n/)
+    .map(line => line.trim())
+    .filter(line => line)
```

Comment:
Does the `.filter()` actually do anything?

Reply:
yes, it's a poor man's check for the empty line here. So that we don't consider empty line as a current dir by default which is unexpected most of the times in a notation like this:
```
files: |

(empty line here)
glob/**
```

Reply:
I had no idea.